lib/string-checker.js

```diff
@@ -83,17 +83,6 @@ StringChecker.prototype = {
      * @throws Error on invalid source code.
      */
     _parse: function(str) {
```

Comment:
OK, we can talk about this another time because it's actually not relevant to the rest of the work you've done here, and I don't want to take attention away from it.